src/community-guides/tips-for-succeeding-at-hackathons.md

```diff
@@ -0,0 +1,5 @@
+1. Build a balanced team with both coders and presenters. Aim for diversity in skills, such as 2 developers and 2 non-developers. Include roles like UI/UX designers and domain experts if relevant. (Anecdote: Won 4 Mumbai-level hackathons with a team of 2 devs and 2 non-devs.)
+2. Select a strong problem statement (PS). Choose one that's likely to reach the final round and can be explained easily. Look for PS with real-world applications but room for improvement. Research experimental PS thoroughly. Treat optional features as mandatory for a winning entry.
+3. Research existing solutions thoroughly. Identify what already exists and what's missing. Add unique features that provide value. Implement all required features and treat optional ones as necessary. Develop unique selling points (USPs) to stand out.
+4. Choose your tech stack wisely. Stick to technologies you're familiar with to maintain productivity. Reuse code from previous projects when allowed. Follow hackathon rules regarding component usage. (Anecdote: Successfully used Pocketbase as a database and backend integration tool in some hackathons, saving significant time.)
```

Comment:
Remove the anecdote, since it's a generic wiki. In place of that give examples on how to utilize boilerplate (where, and how to find them), give comparison on why hosting a db from scratch is not a good idea in comparison to using solutions like pocketbase

Reply:
Ok Sure, will remove anecdotes.